.claude/hookify.no-atlas-rehash.md

```
@@ -0,0 +1,18 @@
+---
+name: no-atlas-rehash
+enabled: true
+event: bash
+pattern: atlas\s+migrate\s+hash
+action: block
+---
+
+**BLOCKED: Do not rehash Atlas migrations.**
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)